onnxruntime/python/tools/transformers/models/whisper/convert_to_onnx.py

```diff
@@ -169,6 +169,78 @@ def parse_arguments(argv=None):
     )
     parser.set_defaults(chain_model=True)
 
+    parser.add_argument(
+        "--use_whisper_beamsearch",
+        required=False,
+        action="store_true",
+        help="When chain_model, using WhisperBeamSearch operator rather than BeamSearch operator. \
+              It will be set to true when collect_cross_qk, extra_decoding_ids or output_no_speech_probs is set.",
+    )
+    parser.set_defaults(use_whisper_beamsearch=False)
+
+    parser.add_argument(
+        "--extra_decoding_ids",
+        required=False,
+        action="store_true",
+        help="Need extra starting decoding ids for some feature like cross qk. Default if false.",
+    )
+    parser.set_defaults(extra_decoding_ids=False)
+
+    parser.add_argument(
+        "--collect_cross_qk",
+        required=False,
+        action="store_true",
+        help="Beam search model collect stacked cross QK.",
+    )
+    parser.set_defaults(collect_cross_qk=False)
+
+    parser.add_argument(
+        "--output_cross_qk",
+        required=False,
+        action="store_true",
+        help="Beam search model output collected qk as output. Also hint collect_cross_qk",
+    )
+    parser.set_defaults(output_cross_qk=False)
+
+    parser.add_argument(
+        "--no_speech_token_id",
+        default=50362,
```

Comment:
Similar 'todo' comment for this token id since the default value may be different between the multilingual and English-only models.